packages/flutter/lib/src/cupertino/slider.dart

```diff
@@ -482,7 +510,9 @@ class _RenderCupertinoSlider extends RenderConstrainedBox implements MouseTracke
         TextDirection.rtl => -valueDelta,
         TextDirection.ltr => valueDelta,
       };
-      onChanged!(_discretizedCurrentDragValue);
+
+      final bool isLargeDelta = valueDelta.abs() > _kLargeDeltaThreshold;
```

Comment:
Delta might not be a good criterion because the rate of update is not fixed. Velocity is more suitable IMO. I recommend recording value and timestamp in `_handleDragStart` and `_handleDragUpdate` to compute the velocity for comparison.